docs/_book.yaml

```diff
@@ -33,6 +33,10 @@ upper_tabs:
       - title: "Constructing Trotter Steps With Low-Rank Decomposition"
         path: /openfermion/tutorials/circuits_3_arbitrary_basis_trotter
 
+    - name: "FQE"
```

Comment:
We generally limit the amount of lower tabs to keep navigation consistent throughout the site. Also, "FQE" isn't very clear title for navigation. I think it would be better to move this a separate section in the existing Guide/Tutorials tab.

Reply:
Could we have it as a separate top tab if it read "OpenFermion-FQE".   Similar to how Cirq has an "Experiments" tab that houses recirq.  My next question is where do the references for this simulator go?  In the "References" tab of OpenFermion?  That doesn't seem right.